dist/common/scripts/scylla_cpuscaling_setup

```diff
@@ -56,6 +56,15 @@ if __name__ == '__main__':
         cpufreq = systemd_unit('cpupower-frequency-set.service')
         cpufreq.enable()
         cpufreq.restart()
+    if is_arch():
+        if not shutil.which('cpupower'):
+            pkg_error_exit('cpupower')
```

Comment:
Can't we use pkg_install here? and consolidate with existing code?

Reply:
Replaced with pkg_install(). It calls pkg_error_exit() on on unsupported distribution anyway, so result is same.